src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,258 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+
+const colorMap = new Map()
+let countClick = 0
+
+//Show description box by pressing left arrow key
+class SeqColor extends VisualizerDefault implements VisualizerInterface {
+    name = 'Sequence Color'
+    n = 40
+
+    params = {
+        n: {
+            value: this.n,
+            forceType: 'integer',
+            displayName: 'number of terms',
+            required: true,
+        },
+    }
+
+    private current_num = 0
+    private X = 0
+    private Y = 0
+    private subG = this.sketch.createGraphics(800, 90)
+    private boxIsShow = false
+    private primeNum: number[] = []
+    private count_prime = 0
```

Comment:
Why is this variable `snake_case` whereas most of your other variables are `camelCase`? For more info on variable naming conventions, see [this Wikipedia article](https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats).